kore/src/main/scala/org/kframework/parser/kore/Interface.scala

```diff
@@ -314,7 +317,8 @@ object Iff {
 }
 
 trait Exists extends Pattern {
-  def s: Sort // this is the sort of the whole exists pattern, not the sort of the binding variable v
+  def s
+      : Sort // this is the sort of the whole exists pattern, not the sort of the binding variable v
```

Comment:
Weird alignment. Maybe if you move the comment on the line above the type annotation will be kept on the same line as the declaration.